### DISCUSSION TITLE
(GitHub Actions) Make `issue_comment_target` trigger available

### DISCUSSION THREAD

**Discussion by epassaro** [community/community#7746] [Actions]
Would be great to have an `issue_comment_target` trigger that works like `pull_request_target`.

**Comment by ethomson** [comment]
Hi @epassaro - can you give me some more information on what you're trying to accomplish?

The difference between `pull_request` and `pull_request_target` is that `pull_request_target` runs in the context of the base of the pull request, rather than the merge commit.  `issue_comment` doesn't really have an analog here, since it always run in the context of the main branch.  So I don't think it's really about the issue comment _target_, per se.

I'd love to have some specific examples of what you're trying to solve?  Thanks!

**Reply by epassaro** [reply]
If `issue_comment` runs always on the context of the main branch, then there's nothing to do. Probably I misunderstood something.

Thanks!
